### PR TITLE
Allow for annotations to be added to deployment

### PR DESCRIPTION
**Pull request by wtmthethird**
Signed-off-by: William Mowery <william@diamanti.com>

#### What this PR does / why we need it:
Allows for annotations to be added to the deployment.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mssql-linux/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.5.1
+version: 0.5.2
```
Comment (davidkarlsen) [inline stable/mssql-linux/Chart.yaml]: new feature - please make it 0.6.0

Comment (wtmthethird) [reply stable/mssql-linux/Chart.yaml]: [done](https://github.com/helm/charts/pull/8478/commits/97b15dca2544a85d54d1d91eaaafd7dc74f7f243), thank you!

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**